include/EffectLabelButton.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * EffectLabelButton.h - class trackLabelButton
+ *
+ * Copyright (c) 2024 Noah Brecht <noahb2713/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_GUI_EFFECT_LABEL_BUTTON_H
+#define LMMS_GUI_EFFECT_LABEL_BUTTON_H
+
+#include <QPushButton>
+
+namespace lmms::gui
+{
+
+class EffectView;
+
+class EffectLabelButton : public QPushButton
+{
+	Q_OBJECT
+public:
+	EffectLabelButton(EffectView* _tv, QWidget* _parent);
+	~EffectLabelButton() override = default;
+
+private:
+	EffectView* m_effectView;
+};
+
+
+} // namespace lmms::gui
+
+#endif // LMMS_GUI_EFFECT_LABEL_BUTTON_H
```

Comment:
Please add a new line at the end of the file. It seems that modern compilers might not need this anymore but I guess it might still be good practice ([link](https://stackoverflow.com/a/72409)).

Reply:
Fixed with commit e6aa0ea051035fa72217ed84c2d862b2618f6e86.